src/lib/mina_base/gen/gen.ml

```diff
@@ -6,7 +6,7 @@ open Core_kernel
 open Signature_lib
 
 let keypairs =
-  let n = 120 in
+  let n = 1200 in
```

Comment:
wow, we need that many?

Reply:
We need more than 120 to ensure enough time between subsequent transactions from the same address

Reply:
This is needed to avoid two transactions from the same address to co-exist in network (as transactions tend to get re-ordered and this is problematic because higher nonce transaction might get discarded)

Reply:
I think as another PR it might be a good idea to just keep secret key files for testing generated exactly once. This way 1200 keys will be less problematic